Source/LuaBridge/detail/Userdata.h

```diff
@@ -59,6 +59,10 @@ class Userdata
 protected:
   void* m_p; // subclasses must set this
 
+  Userdata() : m_p(nullptr)
```

Comment:
Sorry, a couple of thing, again.
There should be spaces around brackets between a letter/digit and a bracket.
And `nullptr` is also a C++11 feature.

Reply:
no problem i fixed your findings